### PR TITLE
Removes greentext from traitors

### PR DESCRIPTION
**Pull request by ynot01**
### Intent of your Pull Request

Discourages the 'win' and gloating mindset
Discourages the validhunt/murderbone to 'win' and 'must redtext the traitor' mindsets
Encourages the addition of more creative and liberal objectives without worrying about how possible it is to track within code
Tested in private server, working with no runtimes
Avant-garde
requesting test merge

#### Changelog

:cl:  
experimental: Removed greentext from traitors 
/:cl:

### CODE REVIEW

**Comment by ToGWtF** [comment]
On the fence.

I like what you hope it will achieve though.

**Comment by PrinceKirze** [comment]
as long as they have objectives i don't see how this will change anything but im always open to seeing how small changes could affect server culture so i'm open to it

**Comment by patpol4** [comment]
Interesting

**Comment by Marmio64** [comment]
is there still objectives to complete, and will they show up on round end? If so this accomplishes nothing. This will still probably not accomplish anything so long as there are objectives to complete.

File: code/modules/antagonists/traitor/datum_traitor.dm [hunk 1]
```diff
@@ -365,33 +365,23 @@
 		var/count = 1
 		for(var/datum/objective/objective in objectives)
 			if(objective.check_completion())
```
Comment (Bibby0110) [inline code/modules/antagonists/traitor/datum_traitor.dm:367]: no need for the if block since both pieces of text are identical

**Comment by boodaliboo** [comment]
why? if your not removing objectives i dont understand

**Comment by ynot01** [comment]
> why? if your not removing objectives i dont understand

i said why

removal of objectives would lead to other things which i don't believe we're ready to tackle yet

File: code/modules/antagonists/traitor/datum_traitor.dm (hunk 1 above)
Comment (ynot01) [reply code/modules/antagonists/traitor/datum_traitor.dm:367]: no they arent

**Comment by Tipy1802** [comment]
Neat, it would allow us to add in more broader objectives that cannot have a complesion check

**Comment by UselessTheremin** [comment]
just because it wont be called greentext or redtext

doesnt mean the problems will go away

in my opinion, we should first think up some fun traitor objectives (possibly steal some from goon) because removing greentext and redtext with the current climate of objectives would do literally nothing as they are so black or white in whether or not you completed them.

**Comment by neeshacark** [comment]
the only thing this will do is make more objectives that couldn't previously be added due to completion check possible

that said I think this is a bad and almost pointless change

**Comment by ghost** [comment]
> Encourages the addition of more creative and liberal objectives without worrying about how possible it is to track within code

you can just make objectives autocomplete instead. that's what we already do for certain antagonists and certain objectives.

removing the win condition doesn't remove objectives-people will still try to complete objectives regardless of if it says they officially win or officially lose, because people like to kill, and killing is all they will generally want to do with some rare exception. so they'll kill what they're allowed to kill or can get away with killing, which is what their objectives serve to restrict them by.

objectives aren't a block preventing you from adding more freeform or creative objectives-they're a leash used to keep murderboners in line. removing the win/lose condition doesn't change that behavior.

File: code/modules/antagonists/traitor/datum_traitor.dm (hunk 1 above)
Comment (Bibby0110) [reply code/modules/antagonists/traitor/datum_traitor.dm:367]: Enlighten me, what’s the difference?

**Comment by jcatjmeow** [comment]
While I do support the removal of greentext I'd like it to look like some other station's freeform objectives and not like this. This is just lazy and sloppy tbh. 

(No offense to you ynot <3)

**Comment by JamieD1** [comment]
Awaiting Vote 751

**Comment by ynot01** [comment]
> While I do support the removal of greentext I'd like it to look like some other station's freeform objectives and not like this. This is just lazy and sloppy tbh.
> 
> (No offense to you ynot <3)

part 1 in branch name, freeform objectives will come through heavy group brainstorming if this gets merged

**Comment by patpol4** [comment]
> > While I do support the removal of greentext I'd like it to look like some other station's freeform objectives and not like this. This is just lazy and sloppy tbh.
> > (No offense to you ynot <3)
> 
> part 1 in branch name, freeform objectives will come through heavy group brainstorming if this gets merged

this, its impossible to make this change without this pr as a transition

**Comment by maxion12345** [comment]
Not sure how this would be liked. Not having win conditions as antagonist might encourage more casual play, but at the same time it feels good to complete your objectives without being caught, and having your name green in a sea of red for the server to see.

**Comment by Sir-Lagsalot** [comment]
I never thought I'd see the day...
And by Ynot, no less

**Comment by fluffe9911** [comment]
Council vote

**Comment by patpol4** [comment]
https://forums.yogstation.net/threads/vote-751-approve-or-deny-this-pr-https-github-com-yogstation13-yogstation-pull-10259-remove-red-and-green-texting.22597/

**Pull request closed without merging**